infer/lib/python/inferlib/capture/util.py

```diff
@@ -24,7 +24,7 @@ def get_build_output(build_cmd):
     #  TODO make it return generator to be able to handle large builds
     proc = subprocess.Popen(build_cmd, stdout=subprocess.PIPE)
     (verbose_out_chars, _) = proc.communicate()
-    return utils.decode(verbose_out_chars).split('\n')
+    return utils.decode(verbose_out_chars).decode('UTF-8').split('\n')
```

Comment:
`utils.decode()` will already call `decode()`, so I have trouble understanding what this is doing. Is there still an issue with the current version of infer? #387 refers to an outdated problem that was fixed (by calling `utils.decode()` here).